src/ResourceManager/DataMigration/Commands.DataMigration/help/New-AzureRmDataMigrationFileShare.md

```diff
@@ -0,0 +1,122 @@
+---
+external help file: Microsoft.Azure.Commands.DataMigration.dll-Help.xml
+Module Name: AzureRM.DataMigration
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.datamigration/New-AzureRmDataMigrationFileShare
+schema: 2.0.0
+---
+
+# New-AzureRmDataMigrationFileShare
+
+## SYNOPSIS
+Creates the FileShare object for the Azure Database Migration Service, which specifies the local network share to take the source database backups to.
+
+## SYNTAX
+
+```
+New-AzureRmDataMigrationFileShare -Path <String> -Credential <PSCredential>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The New-AzureRmDataMigrationFileShare cmdlet creates the FileShare object that specifies the local network share that Azure Database Migration Service can take source database backups to. The service account running source SQL Server instance must have write privileges on this network share.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> New-AzureRmDmsFileShare -Path $fileSharePath -Credential $fileShareCred
```

Comment:
@hitenjava would you mind providing the corresponding output for this cmdlet?

Reply:
Fixed